.github/workflows/ci.yaml

```diff
@@ -446,6 +446,77 @@ jobs:
         run: |
           poetry run pytest tests/ -vvv -n 2 ${PYTEST_OPTS}  ${{ matrix.PYTEST_OPTS }}
 
+  min-btc-support:
+    name: Test BTC version 25.0 ${{ matrix.name }}
+    runs-on: ubuntu-22.04
+    timeout-minutes: 120
+    env:
+      RUST_PROFILE: release  # Has to match the one in the compile step
+      PYTEST_OPTS: --timeout=1200 --force-flaky
+    needs:
+      - compile
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - NAME: clang
+            CFG: compile-clang
+            TEST_DB_PROVIDER: sqlite3
+            COMPILER: clang
+            TEST_NETWORK: regtest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.8
+
+      - name: Install dependencies
+        run: |
+          pip3 install --user pip wheel poetry
+          poetry install
+
+      - name: Download Bitcoin Core
+        run: wget "https://bitcoincore.org/bin/bitcoin-core-25.0/bitcoin-25.0-x86_64-linux-gnu.tar.gz"
```

Comment:
this should be 

        run: wget "https://bitcoincore.org/bin/bitcoin-core-${{ matrix.BITCOIN_VERSION }}/bitcoin-${{ matrix.BITCOIN_VERSION }}-x86_64-linux-gnu.tar.gz"